docker-compose.yml

```diff
@@ -4,7 +4,7 @@ services:
     environment:
       - NODE_ENV=dev
     container_name: server
-    image: goodbuy/login
+    image: goodbuy/nodejs-webserver:jwt-verify-request
```

Comment:
Naming for the image. Please rename it to `goodbuy/nodejs-webserver` until we have a tag var in place.